tests/helper/helper_oc.go

```diff
@@ -766,3 +767,68 @@ func (oc OcRunner) CreateSecret(secretName, secretPass, project string) {
 func (oc OcRunner) GetSecrets(project string) string {
 	return GetSecrets(oc.path, project)
 }
+
+func (oc OcRunner) AddSecret(comvar CommonVar) {
```

Comment:
Please add a comment describing what this function is about.

Reply:
updated with a comment